### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Kilian Froidevaux, Nicolas Bovard

### DISCUSSION THREAD

**Discussion by nicolasbvd** [heig-vd-dai-course/heig-vd-dai-course#465] [Show and tell]
We will implement a Sudoku application where we can do the following: 

- Pick a random grid from a list stored on the server,
- We can play a grid until we specify that we want to quit,
- Client gets feedbacks of wrong inputs and mistakes (invalid moves).

To go further we could implement:
- Versus sudoku race,
- Choose to save or not the current grid.

**Comment by GeraudSilvestri** [comment]
It's good for us

**Comment by nicolasbvd** [comment]
Link repo: https://github.com/nicolasbvd/heig-vd-dai-course-java-tcp-programming-practical-content-sudoku
hash last commit : d203e2d5e1ff71b22af556664234e54cb7623f02
